packit_service/worker/handlers/github_handlers.py

```diff
@@ -154,13 +161,24 @@ def run(self) -> TaskResults:
         for branch in get_branches(
             *self.job_config.metadata.dist_git_branches, default="master"
         ):
-            try:
-                self.api.sync_release(
-                    dist_git_branch=branch, version=self.data.tag_name
-                )
-            except Exception as ex:
-                sentry_integration.send_to_sentry(ex)
-                errors[branch] = str(ex)
+            exception: Optional[Exception] = None
+            for _ in range(RETRY_LIMIT):
+                try:
+                    self.api.sync_release(
+                        dist_git_branch=branch, version=self.data.tag_name
+                    )
+                    exception = None
+                except Exception as ex:
+                    exception = ex
+                    # the archive has not been uploaded to PyPI yet
+                    if FILE_DOWNLOAD_FAILURE in str(ex):
+                        sleep(10)
```

Comment:
What about some constant for this? (And isn't it in seconds?)

Reply:
OK, this is a different interval, but still worth using a constant or providing a comment why we are waiting here.

Reply:
comment added